listenbrainz/webserver/static/js/src/RecentListens.test.tsx

```diff
@@ -832,32 +834,34 @@ describe("Pagination", () => {
   });
 });
 
-describe("pinRecordingModal", () => {
-  it("renders the PinRecordingModal component with the correct props", async () => {
-    const wrapper = mount<RecentListens>(
-      <GlobalAppContext.Provider value={mountOptions.context}>
-        <RecentListens {...props} />
-      </GlobalAppContext.Provider>
-    );
-    const instance = wrapper.instance();
-    const recordingToPin = props.listens[0];
-    let pinRecordingModal = wrapper.find(PinRecordingModal).first();
-
-    // recentListens renders pinRecordingModal with listens[0] as recordingToPin by default
-    expect(pinRecordingModal.props()).toEqual({
-      isCurrentUser: true,
-      recordingToPin: props.listens[0],
-      newAlert: props.newAlert,
-    });
-
-    instance.updateRecordingToPin(recordingToPin);
-    wrapper.update();
-
-    pinRecordingModal = wrapper.find(PinRecordingModal).first();
-    expect(pinRecordingModal.props()).toEqual({
-      isCurrentUser: true,
-      recordingToPin,
-      newAlert: props.newAlert,
-    });
-  });
-});
+// Will re-add this test when feature flag is removed
+
+// describe("pinRecordingModal", () => {
```

Comment:
Is it possible to use `skip` here instead?

Reply:
Changed to skip

Reply:
Thanks!